daemon/cmd/daemon_main.go

```diff
@@ -234,6 +234,9 @@ func init() {
 	flags.Duration(option.ARPPingRefreshPeriod, 5*time.Minute, "Period for remote node ARP entry refresh (set 0 to disable)")
 	option.BindEnv(option.ARPPingRefreshPeriod)
 
+	flags.Bool(option.EnableL2NeighDiscovery, true, "Determines if cilium should perform any L2 neighbor discovery")
```

Comment:
```suggestion
	flags.Bool(option.EnableL2NeighDiscovery, true, "Enables L2 neighbor discovery used by kube-proxy-replacement and IPsec")
```

Reply:
thanks!

Reply:
updated in https://github.com/cilium/cilium/commit/9fb82a4cae8d90464af49760cbef231e3287420b